scripts/vm/network/security_group.py

```diff
@@ -510,7 +510,7 @@ def check_default_network_rules(vm_name, vm_id, vm_ip, vm_ip6, vm_mac, vif, brna
         rules = execute("iptables-save |grep -w %s |grep -w %s |grep -w %s" % (brfw, vif, vmchain_default))
     except:
         rules = None
-    if rules is None or rules is "":
+    if rules is None or rules == "":
```

Comment:
Maybe better to change this to:

<pre>if not rules:</pre>

That captures the None and empty string case right away.

Reply:
good, thanks @wido